test/extended/machines/workers.go

```diff
@@ -0,0 +1,210 @@
+package operators
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/stretchr/objx"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/dynamic"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	machineLabelSelectorWorker = "machine.openshift.io/cluster-api-machine-role=worker"
```

Comment:
not a blocker: we are trying to reduce deps on openshift specific annotations so we could fetch worker machines by matching the associated node label